features/roles.feature

```diff
@@ -20,33 +20,44 @@ Feature: Manage WordPress roles
     When I run `wp role reset author`
     Then STDOUT should be:
       """
-      Success: Reset 0/1 roles.
+      No changes necessary for 'author' role.
+      Success: Roles reset 0/1.
       """
 
     When I run `wp cap remove author read`
     And I run `wp role reset author`
     Then STDOUT should be:
       """
-      Success: Reset 1/1 roles.
+      Restored 10 capabilities to and removed 9 capabilities from 'author' role.
+      Success: Role reset 1/1.
```

Comment:
In case of single role passed as param like `wp role reset author`
You want `Restored 10 capabilities to and removed 9 capabilities from 'author' role.` this message to be skipped and add `Success: Role reset.` ?